ui/components/multichain/global-menu/global-menu.js

```diff
@@ -279,17 +280,17 @@ export const GlobalMenu = ({ closeMenu, anchorElement }) => {
       >
         {t('lockMetaMask')}
       </MenuItem>
-    </Menu>
+    </Popover>
   );
 };
 
 GlobalMenu.propTypes = {
   /**
-   * The element that the menu should display next to
+   * Function to close the menu
    */
-  anchorElement: PropTypes.instanceOf(window.Element),
+  closeMenu: PropTypes.func.isRequired,
   /**
-   * Function that closes this menu
+   * The element to which the menu should be anchored
    */
-  closeMenu: PropTypes.func.isRequired,
+  anchorElement: PropTypes.instanceOf(window.Element),
```

Comment:
Are there any changes here? It seems they are being changed around? Let's revert these changes if it's just changing the ordering as it makes it difficult to see if there have been any updates.